Packages/Tags/Runtime/AtomTags.cs

```diff
@@ -16,6 +16,9 @@ namespace UnityAtoms.Tags
     [AddComponentMenu("Unity Atoms/Tags")]
     public sealed class AtomTags : MonoBehaviour, ISerializationCallbackReceiver
     {
+        // declaring events
+        public event Action<string> TagAdded;
+        public event Action<string> TagRemoved;
```

Comment:
I think It'd be more ideomatic to use AtomEvents here...

Reply:
I have replaced with using AtomEvents there.